instructions_gke.md

```diff
@@ -2,6 +2,64 @@
 
 ### Before you begin
 
+1. Download **Cloud Code source protect (Preview)**
+
+   [Cloud Code](https://cloud.google.com/code/docs) source protect is not available
+   for public access. To get access to this feature, see the
+   [access request page][access].
+
+1. You have two options to get things ready:
+
+
+    - [Use Terraform to get things setup automatically](#Use-Terraform-to-get-things-setup-automatically)
+
+    __or__
+
+    - [Setup the environment manually](#setup-environment-manually)
+
+
+
+
+### Use Terraform to get things setup automatically
+
+1. Setup account info
+
+    -  __(If you don't have an existing project you want to deploy this demo into, skip to the next step)__ If you have an __existing__ project that you want to deploy this demo into, provide it in `deploy-to-gke.tfvars` by un-commenting `existing_project_id`, and replace "sds-java-demo" with the project ID you would like to use.
+
+    __or__
+
+    -  If you want Terraform to __create__ a project for you within your organization's billing account, provide it in `deploy-to-gke.tfvars` by un-commenting `google_billing_account`, and replace "00000-000000-0000" with the billing account number
+
+1. _cd_ into the `tf` folder` and run:
+    ```
+    terraform plan -var-file="deploy-to-gke.tfvars"
```

Comment:
Please add `terraform init` before this step and use `plan -out plan` and `apply "plan"` for both the gke and cloud run instructions.

Reply:
Sounds good!